html/field-studies.html

```diff
@@ -0,0 +1,301 @@
+                    <div><strong>Communities:</strong> {{#expand dcoCommunities}}<a href="{{uri}}" target="_blank">{{name}}</a>{{/expand}}</div>
+                    {{/if}}
+
+                    {{!-- List what portal teams/groups this project is related to, if any --}}
+                    {{#if dcoPortalGroups}}
+                    <div><strong>Portal Groups:</strong> {{#expand dcoPortalGroups}}<a href="{{uri}}" target="_blank">{{name}}</a>{{/expand}}</div>
+                    {{/if}}
+
+                    {{!-- The list of DCO Members and non-DCO Members who participated in this project --}}
+                    {{#if participants}}
+                    <div><strong>Participants:</strong> {{#expand participants}}<a href="{{uri}}" target="_blank">{{name}}</a>{{/expand}}</div>
+                    {{/if}}
+
+                    {{!-- The grants that have funded this project --}}
+                    {{#if grants}}
+                    <div><strong>Grants:</strong> {{#expand grants}}<a href="{{uri}}" target="_blank">{{name}}</a>{{/expand}}</div>
+                    {{/if}}
+
+                    {{!-- BADGES --}}
+
+                    <div>
+                        <a href="{{dcoId}}" target="_blank"><img class="badge" src="{{dcoIdBadgeURL dcoId}}"/></a>
+                    </div>
+
+                </div>
+            </td>
+        </tr>
+    </script>
+
+    <script type="text/javascript">
+        Handlebars.registerHelper('orcidURL', function(orcid) {
+            return "http://orcid.org/"+orcid;
+        });
+        Handlebars.registerHelper('dcoIdBadgeURL', function(dcoId) {
+            var escapedDCOID = encodeURIComponent(dcoId.substring(25));
+            return "//dcotest.tw.rpi.edu/badges/badge?title=metadata&text="+escapedDCOID+"&color=D14719";
+        });
+        Handlebars.registerHelper('showMostSpecificType', function(mostSpecificType) {
+            return (mostSpecificType);
+        });
+        Handlebars.registerHelper('expand', function(items, options) {
+            var out = "";
+            var j = items.length - 1;
+            for(var i = 0; i < items.length; i++) {
+                out += options.fn(items[i]);
+                if(i < j) {
+                    out += "; ";
+                }
+            }
+            return out;
+        });
+        Handlebars.registerHelper('list', function(items, options) {
+            var out = "<ul>";
+            for(var i=0, l=items.length; i<l; i++) {
+                out = out + "<li>" + options.fn(items[i]) + "</li>";
+            }
+            return out + "</ul>";
+        });
+        var source = $("#project-template").html();
+        var template = Handlebars.compile(source);
+    </script>
+
+    <script type="text/javascript">
+        jQuery(document).ready(function($) {
+            $('.facet-view-simple').facetview({
+                search_url: '//localhost:9200/dco/field-study/_search',
+                page_size: 100,
```

Comment:
the arcgis version sets it to 1000. Why not just do that? Or is there a way to set it to all, like maybe 0?